drivers/adc/Kconfig.npcx

```diff
@@ -0,0 +1,12 @@
+# NPCX ADC driver configuration options
+
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config ADC_NPCX
+	bool "Nuvoton NPCX embedd controller (EC) ADC driver"
```

Comment:
```suggestion
	bool "Nuvoton NPCX embedded controller (EC) ADC driver"
```

Reply:
I will add an additional CL for it since I also made the same typo in the other Kconfig.npcx files. Thanks.

Reply:
Done.